draft-ietf-quic-tls.md

```diff
@@ -292,135 +330,164 @@ In {{quic-tls-handshake}}, symbols mean:
   application keys.
 * "{" and "}" enclose messages that are protected by the TLS Handshake keys.
 
-If 0-RTT is not possible, then the client does not send frames protected by the
+If 0-RTT is not attempted, then the client does not send frames protected by the
 0-RTT key (@B).  In that case, the only key transition on the client is from
-cleartext (@A) to 1-RTT protection (@C).
+unprotected packets (@A) to 1-RTT protection (@C), which happens before it sends
+its final set of TLS handshake messages.
 
 The server sends TLS handshake messages without protection (@A).  The server
 transitions from no protection (@A) to full 1-RTT protection (@C) after it sends
 the last of its handshake messages.
 
 Some TLS handshake messages are protected by the TLS handshake record
-protection.  However, keys derived at this stage are not exported for use in
-QUIC.  QUIC frames from the server are sent in the clear until the final
+protection.  These keys are not exported from the TLS connection for use in
+QUIC.  QUIC packets from the server are sent in the clear until the final
```

Comment:
I'm trying to be more consistent about talking about packets when it comes to packet protection.  If I need to talk about frames, it's only to the extent that those frame contain TLS handshake messages.  What is most relevant is either the TLS handshake messages, or the packet.  Frames are sort of sandwiched in the middle there.